spec/components/post_revisor_spec.rb

```diff
@@ -17,13 +18,28 @@
     post = Fabricate(:post, topic: topic)
 
     messages = MessageBus.track_publish("/topic/#{topic.id}") do
-      described_class.new(post).revise!(Fabricate(:admin), { category_id: category.id })
+      described_class.new(post).revise!(admin, { category_id: category.id })
     end
 
     expect(messages.first.data[:refresh_stream]).to eq(nil)
 
     messages = MessageBus.track_publish("/topic/#{topic.id}") do
-      described_class.new(post).revise!(Fabricate(:admin), { category_id: category_solved.id })
+      described_class.new(post).revise!(admin, { category_id: category_solved.id })
+    end
+
+    expect(messages.first.data[:refresh_stream]).to eq(true)
+  end
+
+  it 'sets the refresh option after adding an allowed tag' do
+    SiteSetting.solved_enabled = true
+    SiteSetting.tagging_enabled = true
+    allowed_tag = Fabricate(:tag)
```

Comment:
Can you add a test where the site setting contains multiple tags for which Solved would be enabled?

Reply:
Done!